<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>Program | The Second Korea-Japan Machine Learning Workshop</title>
<link href="../assets/css/all-nocdn.css" rel="stylesheet" type="text/css">
<meta name="theme-color" content="#5670d4">
<meta name="generator" content="Nikola (getnikola.com)">
<link rel="alternate" type="application/rss+xml" title="RSS" hreflang="en" href="../rss.xml">
<link rel="canonical" href="http://aisociety.kr/KJMLW2019/program/">
<!--[if lt IE 9]><script src="../assets/js/html5.js"></script><![endif]--><meta name="author" content="Yung-Kyun Noh">
<meta property="og:site_name" content="The Second Korea-Japan Machine Learning Workshop">
<meta property="og:title" content="Program">
<meta property="og:url" content="http://aisociety.kr/KJMLW2019/program/">
<meta property="og:description" content="Program


Talks
Posters
Participants



Talks



    
    
    
    
    Graph-based Semi-Supervised Learning for Genome, Diseasome, and Drugome
    Hyunjung (Helen) Shin (Ajou University)
    In this">
<meta property="og:type" content="article">
<meta property="article:published_time" content="2018-10-29T18:03:28+09:00">
</head>
<body>
<a href="#content" class="sr-only sr-only-focusable">Skip to main content</a>

<!-- Menubar -->

<nav class="navbar navbar-inverse navbar-static-top"><div class="container">
<!-- This keeps the margins nice -->
        <div class="navbar-header">
            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-navbar" aria-controls="bs-navbar" aria-expanded="false">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="http://aisociety.kr/KJMLW2019/">

                <span id="blog-title">The Second Korea-Japan Machine Learning Workshop</span>
            </a>
        </div>
<!-- /.navbar-header -->
        <div class="collapse navbar-collapse" id="bs-navbar" aria-expanded="false">
            <ul class="nav navbar-nav">
<li>
<a href="#">Program</a>
                </li>
<li>
<a href="../venue/index.html">Venue</a>

                
            </li>
</ul>
<ul class="nav navbar-nav navbar-right"></ul>
</div>
<!-- /.navbar-collapse -->
    </div>
<!-- /.container -->
</nav><!-- End of Menubar --><div class="container" id="content" role="main">
    <div class="body-content">
        <!--Body content-->
        <div class="row">
            
            
<article class="post-text storypage" itemscope="itemscope" itemtype="http://schema.org/Article"><header></header><div class="e-content entry-content" itemprop="articleBody text">
    <div class="section" id="program">
<h2><strong>Program</strong></h2>
<div class="contents local topic" id="contents">
<ul class="simple">
<li><a class="reference internal" href="#talks" id="id1"><strong>Talks</strong></a></li>
<li><a class="reference internal" href="#posters" id="id2"><strong>Posters</strong></a></li>
<li><a class="reference internal" href="#participants" id="id3"><strong>Participants</strong></a></li>
</ul>
</div>
<div class="section" id="talks">
<h3><a class="toc-backref" href="#id1"><strong>Talks</strong></a></h3>
<p>

</p>
<div style="margin-top:10px;">
    <div>
    <img src="../images/hshin.jpg" class="pic">
</div>
    <div>
    <p><strong>Graph-based Semi-Supervised Learning for Genome, Diseasome, and Drugome</strong></p>
    <p>Hyunjung (Helen) Shin (Ajou University)</p>
    <p style="min-height: 170px;">In this talk, we present several applications of graph-based machine learning algorithms to networks of genome, diseasome and drugome. In the genome application, we introduce Gene Ranker which produces scores for genes. It employs graph-based semi-supervised algorithm. On the genome network that is built with protein interaction data and WGCNA data of immune patients, it provides ranks for key genes of immune diseases. In case of diseasome application, we present an approach for disease co-occurrence scoring based on gene-disease-symptom network. For this, the algorithm for hierarchical structure of networks is proposed. The similarity matrix of hierarchical structure is huge, sparse, and tri-diagonal. To make matters worse, the sub-matrices are not necessarily square. Therefore, we propose an algorithm not only alleviates the problem of non-squareness and sparseness but also solves scalability problem.</p>
    </div>
</div>
<div style="margin-top:10px;">
    <div>
    <img src="../images/fcp.jpg" class="pic">
</div>
    <div>
    <p><strong>Riemannian geometry and machine learning for non-Euclidean data</strong></p>
    <p>Frank C. Park (Seoul National University)</p>
    <p style="min-height: 170px;">A growing number of problems in machine learning involve data that is non-Euclidean. A naive application of existing learning algorithms to such data often produces results that depend on the choice of local coordinates used to parametrize the data. At the same time, many problems in machine learning eventually reduce to an optimization problem, in which the objective is to find a mapping from one curved space into another that best preserves distances and angles. We show that these and other problems can be naturally formulated as the minimization of a coordinate-invariant functional that measures the proximity to an isometry of a mapping between two Riemannian manifolds. We first show how to construct general coordinate-invariant functionals of mappings between Riemannian manifolds, and propose a family of functionals that measures how close a mapping is to being an isometry. We then formulate coordinate-invariant distortion measures for manifold learning of non-Euclidean data, and derive gradient-based optimization algorithms that accompany these measures. We also address the problem of autoencoder training for non-Euclidean data using our Riemannian geometric perspective. Both manifold learning and autoencoder case studies involving non-Euclidean datasets illustrate both the underlying geometric intuition and advantages of a Riemannian distortion minimization framework.</p>
    </div>
</div>
<div style="margin-top:10px;">
    <div>
    <img src="../images/skim.png" class="pic">
</div>
    <div>
    <p><strong>Modeling (cancer) cells using multi-omics data</strong></p>
    <p>Sun Kim (Seoul National University)</p>
    <p style="min-height: 170px;">Modeling (cancer) cells using multi-omics data is the ultimate research goal in my lab and we have been making a slow but steady progress over a decade for this goal. In this talk, I will present three of the recently submitted (unpublished) manuscripts towards this goal. (1) sequence level:  Ranked k-spectrum kernel for comparative and evolutionary comparison of exons, introns, and CpG islands, (2) transcript level: Cancer subtype classification and modeling by pathway attention and propagation, and (3) epigenetic level: PRISM: Methylation Pattern-based, Reference-free Inference of Subclonal Makeup.<br> The ranked string kernel paper proposes a new string kernel for comparative and evolutionary genomics. Existing string kernel methods have limitations for comparative and evolutionary studies due to the sensitiveness to over represented $k$-mers when applied on a genome scale. With this bias of over-representations, comparing multiple genomes simultaneously is even more challenging. To address these issues, we propose a novel ranked $k$-spectrum string (RKSS) kernel First, our RKSS kernel utilizes a common $k$-mer set across species, or landmarks, that can be used for comparing arbitrary number of genomes. Second, using landmarks, we can use ranks of $k$-mers rather than $k$-mer frequencies that can produce more reliable distances between genomes. Specifically, RKSS kernel is robust when the $k$-mer pattern is highly biased from repetitive elements or copy number variations as shown in our experiment. To demonstrate the power of RKSS kernel for comparative and evolutionary sequence comparison, we conducted two experiments using 10 mammalian species.<br> Two remaining manuscripts will be presented at this meeting as posters by the first authors. So, I will explain the research problems and the core concepts of our approaches briefly. The cancer pathway attention and propagation paper is to model cancer cells using an ensemble of several hundred deep learning pathway models. Two main ideas for this modeling are: when combining hundred pathway models, we need to catch context-dependent mechanisms of highlighting which pathways are important, thus attention mechanisms; to achieve explainable cancer cell modeling, we used pathway network propagation. The last paper is about PRISM, a computational method to infer cancer sublclones from DNA methylation data.  Each of unknown clone populations are mixed and represented in methylation data and the goal is to decompose the subclonal populations. The main computational challenges are that the number of dimensions is huge, several hundred million, and the data is error prone. We address this daunting problem by utilizing the characteristics of DNA methylation data. First, errors are corrected by modeling methylation patterns of DNMT1 enzyme using HMM. Then, with error-corrected methylation patterns, PRISM focuses on small individual genomic regions, each of which represent the abundance of a subclone. A set of statistics collected from each genomic region is modeled with a beta-binomial mixture. Fitting the mixture with expectation-maximization algorithm finally provides inferred composition of subclones.</p>
    </div>
</div>


</div>
<div class="section" id="posters">
<h3><a class="toc-backref" href="#id2"><strong>Posters</strong></a></h3>
<p>

</p>
<ol>
<li class="poster-list">
<b>TBD</b><br>
Ryuichiro Hataya (RIKEN-AIP)
</li>
<li class="poster-list">
<b>Universality of localized explosive transport in magnetically confined plasma confirmed by deep learning technique</b><br>
Gunsoo Yoon (POSTECH)
</li>
<li class="poster-list">
<b>TBD</b><br>
Chao Li (RIKEN-AIP)
</li>
<li class="poster-list">
<b>Riemannian Distortion Measures for Non-Euclidean Data</b><br>
Cheongjae Jang (Seoul National University)
</li>
<li class="poster-list">
<b>TBD</b><br>
Tanuj Kr Aasawat (RIKEN-AIP)
</li>
<li class="poster-list">
<b>Kullback-Leibler Divergence Estimation using Variationally Weighted Kernel Density Estimators</b><br>
Sangwoong Yoon (Seoul National University)
</li>
<li class="poster-list">
<b>TBD</b><br>
Ming Hou (RIKEN-AIP)
</li>
<li class="poster-list">
<b>Scene graph generation using graph based neural networks</b><br>
Yu-Jung Heo (Seoul National University)
</li>
<li class="poster-list">
<b>TBD</b><br>
Tsuyoshi Okita (RIKEN-AIP)
</li>
<li class="poster-list">
<b>TBD</b><br>
Takeshi Teshima (RIKEN-AIP)
</li>
<li class="poster-list">
<b>TBD</b><br>
Yung-Kyun Noh (Seoul National University)
</li>
<li class="poster-list">
<b>On local optimizers of acquisition functions in Bayesian optimization</b><br>
Jungtaek Kim (POSTECH)
</li>
<li class="poster-list">
<b>TBD</b><br>
Runa Eschenhagen (RIKEN-AIP)
</li>
<li class="poster-list">
<b>Set transformer: A framework for attention-based permutation-invariant neural networks</b><br>
Yoonho Lee (KAKAO)
</li>
<li class="poster-list">
<b>TBD</b><br>
Haixia Zheng (RIKEN-AIP)
</li>
<li class="poster-list">
<b>Cancer cell modeling by deep pathway attention and propagation</b><br>
Sangseon Lee (Seoul National University)
</li>
<li class="poster-list">
<b>On comparison of dynamical systems via reproducing kernel Hilbert spaces</b><br>
Isao Ishikawa (RIKEN-AIP)
</li>
<li class="poster-list">
<b>PRISM: Methylation Pattern-based, Reference-free Inference of Subclonal Makeup</b><br>
Dohoon Lee (Seoul National University)
</li>
<li class="poster-list">
<b>The global optimum is attained by ridgelet transform</b><br>
Masahiro Ikeda (RIKEN-AIP)
</li>
<li class="poster-list">
<b>Sequential Structure Learning with Temporal Dependency Networks for Video Understanding</b><br>
Kyoung Woon On (Seoul National University)
</li>
<li class="poster-list">
<b>Synthesis of earthquake ground motions using embedding and neural networks</b><br>
Tomohisa Okazaki (RIKEN-AIP)
</li>
<li class="poster-list">
<b>Motion planning based reinforcement learning</b><br>
Jinwon Choi (Seoul National University)
</li>
<li class="poster-list">
<b>Beta Bernoulli dropout</b><br>
Saehoon Kim (AITRICS)
</li>
<li class="poster-list">
<b>TBD</b><br>
Francis Hamilton (RIKEN-AIP)
</li>
<li class="poster-list">
<b>Efficient Neural Network Compression via Transfer Learning for Industrial Optical Inspection</b><br>
Seunghyeon Kim (Seoul National University)
</li>
<li class="poster-list">
<b>TBD</b><br>
Jongyeong Lee (RIKEN-AIP)
</li>
<li class="poster-list">
<b>Applying Deep Reinforcement Learning to One-on-one Fighting of a Complex Commercial Game 'Blade and Soul'</b><br>
Seong Ho Son (NC Soft)
</li>
<li class="poster-list">
<b>Persistence Fisher Kernel: A Riemannian Manifold Kernel for Persistence Diagrams</b><br>
Tam Le (RIKEN-AIP)
</li>
</ol>
</div>
<div class="section" id="participants">
<h3><a class="toc-backref" href="#id3"><strong>Participants</strong></a></h3>
<p>The list of participants will be updated soon.</p>
</div>
</div>
    </div>
    

</article>
</div>
        <!--End of body content-->

        <footer id="footer">
            Contents © 2019         <a href="mailto:yungkyun.noh@gmail.com">Yung-Kyun Noh</a> - Powered by         <a href="https://getnikola.com" rel="nofollow">Nikola</a>         
            
        </footer>
</div>
</div>


            <script src="../assets/js/all-nocdn.js"></script><!-- fancy dates --><script>
    moment.locale("en");
    fancydates(0, "YYYY-MM-DD HH:mm");
    </script><!-- end fancy dates --><script>
    baguetteBox.run('div#content', {
        ignoreClass: 'islink',
        captions: function(element) {
            return element.getElementsByTagName('img')[0].alt;
    }});
    </script>
</body>
</html>
